lib/network/level-net/utils.js

```diff
@@ -0,0 +1,30 @@
+'use strict'; // eslint-disable-line
+
+module.exports.serializeError = function serializeError(err) {
+    if (!err) {
+        return err;
+    }
+    const serializedErr = Object.assign({}, err);
+
+    serializedErr.message = err.message;
+    for (const k in err) {
+        if (!(k in serializedErr)) {
+            serializedErr[k] = err[k];
+        }
+    }
```

Comment:
I think we want to copy a new error since it's going to be a separate serialized object.